src/components/Lease/Authentication/Content.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import AppBar from '@mui/material/AppBar';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import Paper from '@mui/material/Paper';
+import Grid from '@mui/material/Grid';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Tooltip from '@mui/material/Tooltip';
+import IconButton from '@mui/material/IconButton';
+import SearchIcon from '@mui/icons-material/Search';
+import RefreshIcon from '@mui/icons-material/Refresh';
+import InventoryTable from '../../Table/Table';
+import ResourceDatagrid from '../../Datagrid/ResourceDatagrid';
+
+export default function Content() {
```

Comment:
do we need both .js and .tsx? they seem to be the same

Reply:
We do not, I can remove the typescript files and just do the open API bindings in typescript.